CHANGELOG/v1.6.2.md

```diff
@@ -0,0 +1,43 @@
+## 👌 Kubernetes version support
+
+- Management Cluster: v1.25.x -> v1.29.x
+- Workload Cluster: v1.23.x -> v1.29.x
+
+[More information about version support can be found here](https://cluster-api.sigs.k8s.io/reference/versions.html)
+
+## Changes since v1.6.1
+## :chart_with_upwards_trend: Overview
+- 16 new commits merged
+- 3 bugs fixed 🐛
+
+## :bug: Bug Fixes
+- ClusterCacheTracker: Fix ClusterCacheTracker memory leak (#10064)
+- Machine: Watch external objects for machine before deleting (#10177)
+- MULTIPLE_AREAS[API/e2e]: Restore v1alpha3/v1alpha4 conversion to fix SSA issue & add e2e test coverage (#10151)
+
+## :seedling: Others
+- clusterctl: Bump cert-manager to 1.14.2 (#10127)
+- clusterctl: Bump cert-manager to v1.14.1 (#10120)
```

Comment:
where did we land on consolidating these?  i think they can be collapsed into 1 line?